berty-bridge-expo/example/README.md

```diff
@@ -0,0 +1,35 @@
+# berty-bridge-expo
+
+This is the bertybridge Expo module.
+
+## Building the Expo module example app
+
+Open a terminal in this folder and run:
+
+```
+cd .. # cd to the parent folder
+make ios.gomobile
+npm run build
+npm run build plugin
```

Comment:
I am able to build and run without `npm run build` and `npm run build plugin`. Maybe mention that these are only needed for development.